projects/packages/masterbar/src/admin-menu/class-base-admin-menu.php

```diff
@@ -540,11 +540,6 @@ public function is_item_visible( $item ) {
 	 * Adds a dashboard switcher to the list of screen meta links of the current page.
 	 */
 	public function add_dashboard_switcher() {
-		static $is_added = false;
-		if ( $is_added ) {
-			return;
-		}
```

Comment:
You might want to remove the same fix from the `dashboard_switcher_scripts` function

https://github.com/Automattic/jetpack/blob/3068f8b078164bcfedd45271bf403f7cbc837b6d/projects/packages/masterbar/src/admin-menu/class-base-admin-menu.php#L575-L578

Reply:
Oh, that's right, thanks!